Makefile

```diff
@@ -232,20 +247,42 @@ update-codegen:
 	$(call run-informer-gen,)
 .PHONY: update-codegen
 
-# $1 - target file
-define generate-crd
-	$(CONTROLLER_GEN) crd paths="$(GO_PACKAGES)" output:crd:stdout > '$(1)'
+verify-codegen:
+	$(call run-deepcopy-gen,--verify-only)
+	$(call run-client-gen,--verify-only)
+	$(call run-lister-gen,--verify-only)
+	$(call run-informer-gen,--verify-only)
+.PHONY: verify-codegen
+
+# $1 - api package
+# $2 - output dir
+# We need to cleanup `---` in the yaml output manually because it shouldn't be there and it breaks opm.
+define run-crd-gen
+	$(CONTROLLER_GEN) crd paths='$(1)' output:dir='$(2)'
+
 endef
 
-update-crd:
-	$(call generate-crd,$(CRD_PATH))
-.PHONY: update-crd
+# $1 - dir prefix
+define generate-crds
+	$(foreach p,$(api_package_dirs),$(call run-crd-gen,$(p),$(1)$(p)))
+	find $(foreach p,$(api_package_dirs),$(1)$(p)) -mindepth 1 -maxdepth 1 -type f -name '*.yaml' -exec $(YQ) -i eval '.' {} \;
```

Comment:
shouldn't this be part of verify-crds?

Reply:
oh, this is needed to reformat output files in yq formatting style right?

Reply:
yep, it looses the `---`

Reply:
yep, but it looks like the comment got left in the other macro

Reply:
I've moved the find to the comment